docs/architecture/cloud-native/combine-containers-serverless-approaches.md

```diff
@@ -1,47 +1,40 @@
 
 ## When does it make sense to use containers with serverless?
 
-By default, your serverless functions have no knowledge of the platform on which they'll run. However, in some cases you may have specific requirements that make it important for you to customize the container image in which your code will run. You may want to use a custom image because your function relies on a specific language version or has dependencies or configuration requirements that aren't supported by the default image. In these cases, it may make sense to customize your own container and deploy your function in a custom Docker container.
+Your Azure Function has no knowledge of the platform on which it's deployed. For some scenarios, you may have specific requirements and need to customize the environment on which your function code will run. You'll need a custom image that supports dependencies or a configuration not supported by the default image. In these cases, it makes sense to deploy your function in a custom Docker container.
 
 ## When should you avoid using containers with Azure Functions?
 
-If you're expecting to benefit from the consumption plan billing for your function, you won't be able to do so if you're using your own container. What's more, if you go beyond just using a Docker container and deploy your functions to your own Kubernetes cluster, you'll no longer benefit from the built-in scaling provided by Azure Functions. You'll need to use Kubernetes' scaling features, described below.
+If you want to use consumption billing, you won't be able to run your function in a container. What's more, if you deploy your function to a Kubernetes cluster, you'll no longer benefit from the built-in scaling provided by Azure Functions. You'll need to use Kubernetes' scaling features, described earlier in this chapter.
 
 ## How to combine serverless and Docker containers
 
-To wrap an Azure Function in a Docker container, install the Azure Functions Core Tools and then run the following command:
+To wrap an Azure Function in a Docker container, install the [Azure Functions Core Tools](https://github.com/Azure/azure-functions-core-tools) and then run the following command:
 
 ```console
-func init ProjectName --docker
+func init ProjectName --worker-runtime dotnet --docker
 ```
 
-Choose which worker runtime you want from the following options:
+Once created, the function will include a Dockerfile. You can create and test your function locally. When deploying a function in containers, the ritual of building an image and hosting a container isn't necessary. You write your function code, include a Docker file, and trigger the function. It deploys without additional overhead or configuration.
```

Comment:
```suggestion
When the project is created, it will include a Dockerfile and the worker runtime configured to `dotnet`. Now, you can create and test your function locally. Build and run it using the  `docker build` and `docker run` commands. For detailed steps to get started building Azure Functions with Docker support, see the [Create a function on Linux using a custom image](https://docs.microsoft.com/azure/azure-functions/functions-create-function-linux-custom-image) tutorial.
```